keyboards/planck/rev7/keymaps/vial/vial.json

```diff
@@ -0,0 +1,108 @@
+{
+    "name": "DROP Planck rev7",
+    "vendorId": "0x03A8",
+    "productId": "0xA4F9",
+    "keycodes": ["qmk_lighting"],
+    "menus": ["qmk_rgblight"],
```

Comment:
Sure, I took this from the rev6.1 on the via repository, since they have the same product and vendor ids. Didn't change it at all, but will fix it up soon